### ISSUE TITLE
UI: Metadata in search results

### ISSUE THREAD

**Issue by Vort** [ortexx/museria-global#6]
It would be useful if search result contains such information as bitrate, duration or file size.
For example, user may want to upgrade his 128 kbps file to 320 kbps one.
Or user may have short version of the song and want to download full version (compare duration).

![sr](https://user-images.githubusercontent.com/1242858/78242258-58d8e480-74ea-11ea-8db8-bae2e37f8a63.png)

**Comment by ortexx** [comment]
Yes, i have an idea to improve this section in the future. Now you can open it in you browser and see the duration or play/download the file.

**Comment by ortexx** [comment]
https://github.com/ortexx/museria/issues/5
